src/labkey/filter/Types.ts

```diff
@@ -42,22 +42,24 @@ export interface IFilterType {
      */
     getURLParameterValue: (value: FilterValue) => FilterValue
     validate: (value: FilterValue, jsonType: string, columnName: string) => any
+
+    getLabKeySqlOperator: () => string
```

Comment:
nit: Consider adding brief documentation for this new method as this will show up on https://labkey.github.io/labkey-api-js/interfaces/Filter.IFilterType.html